self_teach_challenge_1.1/styles.css

```diff
@@ -0,0 +1,64 @@
+* {
+    font-size: 10px;
+    font-family:'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+.input_div {
+    display: flex;
+    align-items: center;
+}
+
+h1 {
+    display: inline-block;
+    font-size: 7rem;
+    color: #E94B3CFF;
+}
+
+body {
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    margin: auto;
+    justify-content: center;
+    align-items: center;
+    background: radial-gradient(circle, #E94B3CFF, #2D2926FF);
```

Comment:
Do: These hex codes seem too long, it should be 6 characters (2 for each of red, green, and blue)